### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by keladhruv**
**What**
Update the code-generator, client-go dependencies 

**Why**
[client go changes](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.18.md#client-go)

### CODE REVIEW

**Review by bojand — APPROVED** [review]

**Comment by keladhruv** [comment]
I accidentally approved it before getting approval from you as well @wanjunlei , @benjaminhuo 

Please let me know if you need me to make any changes/revert the commit.

**Comment by benjaminhuo** [comment]
@keladhruv Thanks for this update! It looks fine!

Ben

**Pull request merged**
